claim_ai_quality/utils.py

```diff
@@ -0,0 +1,41 @@
+from claim.models import Claim
+from django.db.models import Q
+
+
+def get_rejected_claim_json_extension(claim):
+    return {
+        "was_categorized": False,
+        "request_time": str(claim.validity_from),
+        "response_time": str(claim.validity_from)
+    }
+
+def get_base_claim_ai_json_extension():
+    return {
+        "was_categorized": False,
+        "request_time": "None",
+        "response_time": "None"
+    }
+
+
+def add_json_ext_to_all_submitted_claims():
+    all_submitted_claims = Claim.objects\
+        .filter(
+            status__in=(Claim.STATUS_CHECKED, Claim.STATUS_REJECTED),
+            validity_to__isnull=True)\
```

Comment:
We need to add also the filter for json_ext doesn't have the claim_ai_quality entry. After the AI and manual review the claim is still in checked or moves to rejected status and that changes after processing the claims. So we risk to rewrite the json_ext entry.

Reply:
@dragos-dobre Currently there's no possibility for filtering  content of json_ext directly by json keys, I added filtering based on the content of the value casted to the string.